bin/vscode-setup

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env bash
+
+# DOC: Open an sbt CLI in the dev environment container.
```

Comment:
issue (blocking): doc string is inaccurate

Reply:
Done